src/components/alarm/AlarmController.cpp

```diff
@@ -15,10 +15,13 @@
     You should have received a copy of the GNU General Public License
     along with this program.  If not, see <https://www.gnu.org/licenses/>.
 */
-#include "AlarmController.h"
+#include "components/alarm/AlarmController.h"
```

Comment:
Do these cpp file specific headers also need to be non relative? Normally it's pretty safe to assume they're in the same directory.

Reply:
not strictly necessary, just consistent

Reply:
I would personally rather not type the entire path for these cases as it seems unnecessary. For everything else it's fine.

Reply:
@geekbozu @Riksu9000 what's your stance on this?

- :+1: for consistently using `#include "components/alarm/AlarmController.h"`
- :-1: for using the shorter `#include "AlarmController.h"` (IMHO looks inconsistent as soon as a include from the same component is directly after the main header include)

Reply:
I have been trying to think about which would be better, but honestly I'm fine with either. If no one has anything to add, then we can just leave it as it is.

Reply:
I don't think we'll end up with name conflicts, unless it's something generic. But I guess it wouldn't hurt to use the full path

Reply:
I'm a fan of verbosity, while it is a pain to have them long...its only a few more keystrokes for explicitness, And explicit saves headache later.